charts/backstage/values.yaml

```
@@ -81,26 +81,38 @@ upstream:
         cpu: 1000m
         memory: 2.5Gi
         ephemeral-storage: 5Gi
+    startupProbe:
+      # This gives enough time upon container startup before the liveness and readiness probes are triggered.
+      # Giving (120s = initialDelaySeconds + failureThreshold * periodSeconds) to account for the worst case scenario.
+      httpGet:
+        path: /.backstage/health/v1/liveness
+        port: backend
+        scheme: HTTP
+      initialDelaySeconds: 60
```

```suggestion
      initialDelaySeconds: 30
```

After testing with @PatAKnight and @gustavolira , we think we can reduce this.